src/frontend/komponenter/Fagsak/InstitusjonOgVerge/SamhandlerTabell.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import type { ISamhandlerInfo } from '../../../typer/samhandler';
+
+export const SamhandlerTabell: React.FC<{ samhandler: ISamhandlerInfo }> = ({ samhandler }) => {
+    return (
+        <>
+            <table className="tabell">
```

Comment:
Bruk nye Table: https://aksel.nav.no/designsystem/komponenter/table
Den er tatt i bruk andre steder i ba-sak-frontend, så titt rundt i koden for eksempler :)